index.js

```diff
@@ -1,4 +1,13 @@
 'use strict'
 
-const { sagaTestEngine } = require('./src')
-module.exports = sagaTestEngine
+const {
+    collectPuts,
+    collectCalls,
+    collectCallsAndPuts,
+    createSagaTestEngine
+} = require('./src')
+
+exports.default = collectPuts
+exports.collectCalls = collectCalls
+exports.collectCallsAndPuts = collectCallsAndPuts
+exports.createSagaTestEngine = createSagaTestEngine
```

Comment:
I prefer the name `createSagaTestEngine` over something more specific since it would open the opportunity to add more configuration options in the future without having to break the API again